internal/db/schema/migrations/oss/postgres/60/01_ldap.up.sql

```diff
@@ -326,6 +358,14 @@ create table auth_ldap_account (
       check(lower(trim(login_name)) = login_name)
     constraint login_name_must_not_be_empty
       check(length(trim(login_name)) > 0),
+  email wt_email, 
+  full_name wt_full_name, 
+  dn text -- will be null until the first successful authentication
+    constraint dn_must_not_be_empty
+      check(length(trim(dn)) > 0), 
+  member_of_groups jsonb -- will be null until the first successful authentication
+    constraint member_of_groups_must_not_be_empty
+      check(length(trim(member_of_groups::text)) > 0),
```

Comment:
Is the structure of this known or completely freeform?

Reply:
it's a json array.

Reply:
Are we going to be parsing this data anywhere in the application or just retrieve it when authenticating and returning it in some API calls? I'm asking because I would feel better about this being a `text[]` if we do need to use the individual values in our application, since it would take any potential guesswork of structure out of the equation.

Reply:
there are many reasons to make it jsonb over text based on how it's going to be used in the database when vivifying LDAP managed groups.

The LDAP repo functions use `[]string` for input/output of an account's groups and the service API and CLI will follow that pattern as well.  That means Boundary in general will just see a slice of an account's groups and never have any idea how it's stored.